src/Kunstmaan/FixturesBundle/Builder/PageBuilder.php

```diff
@@ -229,4 +248,25 @@ private static function incrementString($string, $append = '-v')
             return $string . $append . '1';
         }
     }
+
+    /**
+     * @param PageInterface $parentPage
+     * @param string $childPageClass
+     * @return bool
+     */
+    private function canHaveChild(PageInterface $parentPage, $childPageClass)
+    {
+        $list = array_merge(
+            $this->pagesConfiguration->getPossibleChildTypes(get_class($parentPage)),
+            $parentPage->getPossibleChildTypes()
+        );
```

Comment:
`PagesConfiguration` will call `Page::getPossibleChildTypes` if nothing is set in the configuration. You cannot add/override using this method if a config is present (this is the behaviour when adding new pages via Admin Panel). This way you could skip the `$parentPageMock` altogether.